### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by fbartels** [telephoneorg/dcgoss#1]
Hi,

I've found this project because I wanted to use goss for a project that utilised docker-compose. While the wrapper itself still seems to be working the docker approach described hasn't been updated in quite a while and therefore uses outdated Docker and docker-compose. (also I would think you could slim this down quite a bit when the alpine based docker:18.09.1 as a base image).

Before starting to rewrite everything I wanted to check if this project is still alive and would accept a pull request.

PS: there is also a duplicate copy of this code located at https://github.com/joeblackwaslike/dcgoss. maybe this should reference this as the "upstream"?

**Comment by fbartels** [comment]
Opened pr https://github.com/telephoneorg/dcgoss/pull/2

**Comment by fbartels** [comment]
For others stumbling upon this repository. I have refactored dcgoss a bit and it has meanwhile been merged into the goss repository itself. It can be found at https://github.com/aelsabbahy/goss/tree/master/extras/dcgoss

**Comment by dakky** [comment]
any updates? is this project still alive?

**Comment by fbartels** [comment]
I think this specific repo is pretty much dead. But the script and some additional changes have been merged into the upstream goss repo for quite some time now.

**Comment by dakky** [comment]
hmm i think about forking and changing the way it works. imho its quite useless to have a docker-compose setup configured, but to startup only a single service for testing. 
the advantage is the possibility to start up a database as requierement for a service and run the goss tests in this service.

**Comment by dakky** [comment]
ah i see what u mean ... dcgoss is part of the goss repo ... i'll adress my issue there ;) thanks

**Comment by fbartels** [comment]
> dcgoss is part of the goss repo ...

exactly. I've linked to it already in https://github.com/telephoneorg/dcgoss/issues/1#issuecomment-564243644
